m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh

```diff
@@ -217,7 +217,7 @@ bulkio_test()
 
 show_write_speed()
 {
-	cat $MOTR_TEST_LOGFILE | grep copied | tail -2 | head -1 | \
+	cat "$MOTR_TEST_LOGFILE" | grep copied | tail -2 | head -1 | \
```

Comment:
Maybe this Codacy warning can be fixed, too.

Reply:
done